### PR TITLE
Update zh readme

### PR DESCRIPTION
**Pull request by lunny**
Follow the English Readme

### CODE REVIEW

File: README_ZH.md [hunk 1]
```diff
@@ -8,121 +8,30 @@
...
-- [BearyChat](https://bearychat.com/) (Team Communication)
-- [HiWork](http://www.hiwork.cc/) (Team Communication)
-
-### 产品支持
-
-- [Synology](https://www.synology.com) (Docker)
-- [One Space](http://www.onespace.cc) (App Store)
+1. **开始贡献代码之前请确保你已经看过了 [贡献者向导（英文）](CONTRIBUTING.md)**.
+2. 所有的安全问题，请私下发送邮件给 **security@gitea.io**。谢谢！
+3. 如果你要使用API，请参见 [API 文档](https://godoc.org/github.com/go-gitea/go-sdk).
```
Comment (tboerger) [inline README_ZH.md:30]: Please append `/gitea` to the URL, the english link is broken, https://github.com/go-gitea/gitea/issues/461

Comment (lunny) [reply README_ZH.md:30]: I used https://godoc.org/code.gitea.io/sdk/gitea

**Comment by andreynering** [comment]
LGTM

File: README_ZH.md (hunk 1 above)
Comment (tboerger) [reply README_ZH.md:30]: Will be fixed with https://github.com/go-gitea/gitea/pull/463

**Comment by tboerger** [comment]
Trusted LGTM

**Pull request merged**